cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/r4/R4CareGapStatusEvaluator.java

```diff
@@ -75,4 +94,17 @@ public CareGapsStatusCode getGapStatus(Measure measure, MeasureReport measureRep
 
         return CareGapsStatusCode.CLOSED_GAP;
     }
+
+    /*
+    // TODO implement Measure Group Level improvement notation extension
```

Comment:
Do you plan to address this TODO?

Reply:
yes, this is the same related feature that will be addressed separately from this merge.